src/modules/fancyzones/lib/ZoneSet.cpp

```diff
@@ -77,17 +77,33 @@ IFACEMETHODIMP ZoneSet::RemoveZone(winrt::com_ptr<IZone> zone) noexcept
 
 IFACEMETHODIMP_(winrt::com_ptr<IZone>) ZoneSet::ZoneFromPoint(POINT pt) noexcept
 {
+    winrt::com_ptr<IZone> smallestKnownZone = nullptr;
     for (auto iter = m_zones.begin(); iter != m_zones.end(); iter++)
     {
         if (winrt::com_ptr<IZone> zone = iter->try_as<IZone>())
         {
-            if (PtInRect(&zone->GetZoneRect(), pt))
+            RECT* newZoneRect = &zone->GetZoneRect();
+            if (PtInRect(newZoneRect, pt))
             {
-                return zone;
+                if(smallestKnownZone == nullptr)
```

Comment:
As a rule of thumb, let's use the original file formatting style. So please add a space after `if`. Thanks.